### PR TITLE
[UXE-5724] feat: lazy load Edge App/Edge Firewall on Add/Edit Domain screen

### PR DESCRIPTION
**Pull request by lucasmendes21**
### What is the new behavior introduced by this PR?
implement lazy loader dropdown

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
https://jam.dev/c/25cf77c4-2019-464d-888c-a2d3c7e4fac2

### Does it have a link on Figma?
No
<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [ ] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: src/services/edge-application-services/v4/load-edge-application-service.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { AxiosHttpClientAdapter, parseHttpResponse } from '@/services/axios/AxiosHttpClientAdapter'
+import { makeEdgeApplicationV4BaseUrl } from './make-edge-application-v4-base-url'
+
+export const loadEdgeApplicationsService = async (globalId) => {
+  let httpResponse = await AxiosHttpClientAdapter.request({
+    url: `${makeEdgeApplicationV4BaseUrl()}/${globalId}?fields=id,name`,
```
Comment (pauloSF0) [inline src/services/edge-application-services/v4/load-edge-application-service.js:6]: Só uma dúvida mesmo, não deveria ter os resto das querys aqui ? não vi o page_size e page

Comment (lucasmendes21) [reply src/services/edge-application-services/v4/load-edge-application-service.js:6]: nesse caso não porque é um get e não uma listagem.

Comment (pauloSF0) [reply src/services/edge-application-services/v4/load-edge-application-service.js:6]: não é sobre a paginação no dropdown ?

**Review by pauloSF0 — DISMISSED** [review]

**Comment by lucasmendes21** [comment]
link para validação -> https://rr7tzicysz.map.azionedge.net

**Review by pauloSF0 — APPROVED** [review]

**Review by peterpaulo-azion — DISMISSED** [review]

**Review by HerbertJulio — DISMISSED** [review]

**Review by pauloSF0 — DISMISSED** [review]

**Review by peterpaulo-azion — APPROVED** [review]

**Pull request merged**